etc/kayobe/kolla/config/prometheus/system.rules

```diff
@@ -24,6 +24,24 @@ groups:
       summary: "Prometheus exporter at {{ $labels.instance }} reports low memory"
       description: "Available memory is {{ $value }} GiB."
 
+  - alert: LowSwapSpace
+    expr: (node_memory_SwapFree_bytes / node_memory_SwapTotal_bytes) < {% endraw %}{{ alertmanager_node_free_swap_warning_threshold_ratio }}{% raw %}
```

Comment:
What happens when there is no swap, i.e. node_memory_SwapTotal_bytes is 0?

Reply:
That's good point. It seems like either adding conditional templating or updating ``expr`` for these are needed. @markgoddard  How should I do it? Do you want me to revert this merge and re-do the PR?

Reply:
It depends on how long it will take for you to test it out and provide a fix. If it can be done quickly then no need to merge.

Reply:
The divide by zero evaluates to NaN, and the alert appears as 'OK'. Context: https://www.robustperception.io/get-thee-to-a-nannary/

Reply:
Example: 
![image](https://github.com/stackhpc/stackhpc-kayobe-config/assets/1140727/ece9fb34-37a6-4d82-9f6d-da699f4759bd)

Reply:
So I think this alert is fine as it is, but a good question to ask none-the-less.